docs/model_configuration.md

```diff
@@ -501,6 +501,48 @@ the device kind of the instance, for instance, KIND_CPU is "cpu", KIND_MODEL is
   ]
 ```
 
+### Rate Limiter Config
+
+Instance group optionally specify rate limiter config which controls
+how [rate limiter](rate_limiter.md) operates on instances in the group.
+The rate limiter configuration includes the following specification:
+
+#### Resources
+
+The resources required to execute the request on a model instance.
+Resources are just names with corresponding count. They are replicated
```

Comment:
'resources' are a mapping of a name for the corresponding resource and their corresponding counts.